identity-service/src/config.js

```diff
@@ -235,12 +241,24 @@ const config = convict({
     env: 'minimumBalance',
     default: null
   },
+  minimumRelayerBalance: {
+    doc: 'Minimum token balance for relayer below which /balance_check fails',
+    format: Number,
+    env: 'minimumRelayerBalance',
+    default: null
```

Comment:
lets set this to the same default value to all the minimums here. like 1?

Reply:
Tested this locally to verify that the defaults work.